src/client/src/ui/spotTile/views/tradeNotification.jsx

```diff
@@ -30,7 +30,7 @@ export default class TradeNotification extends React.Component {
         'trade-notification',
         className,
         {
-          'trade-notification--rejected': tradeExecutionNotification.status === TradeStatus.Rejected
+          'trade-notification--rejected': tradeExecutionNotification.status === TradeStatus.Rejected.name
```

Comment:
This is the downside of using static props. We should have a discussion on this pattern.